### ISSUE TITLE
Peculiarity with captured MCV in RA2 mode.

### ISSUE THREAD

**Issue by CCCP84** [CnCNet/cncnet-yr-client-package#302]
The MCV captured by Yuri in the original RA2 can be deployed. In CNCNET - no.
If this can be fixed, then you need to make sure that MCV, forcibly captured from an ally, cannot be deployed. Or better to make it impossible to capture an ally's MCV. 
 
(This is not a big problem)

**Comment by CnCRAZER** [comment]
@CCCP84 can you check if this issue still persists with ares / Phobos + new spawner on dev build?

**Comment by CCCP84** [comment]
Yes, this feature remains

**Comment by CnCRAZER** [comment]
In my personal testing I've observed that mind control'd mcvs can deploy as long as it's into a buildingtype that's not considered a construction yard. Hmm

**Comment by CCCP84** [comment]
I didn't quite understand you. Can they be deployed in your tests or not? 
I don't know how your tests might differ from mine, they can't be deployed. 
Although in the original ra2 it was possible.

By the way, I'm testing this and I came across another bug. Perhaps it's again related to Neutral House. But if you capture and release MCV twice with Yuri, an error occurs. *only in YR mode.
We need to test this in a game with another player

**Comment by CCCP84** [comment]
Yes, I tested this with another player, and no error occurs. It's probably the same thing as with the neutral engineer. When you capture a neutral's MCV and then release it, the neutral tries to deploy it, and an error occurs.

**Comment by CnCRAZER** [comment]
> I didn't quite understand you. Can they be deployed in your tests or not? 
> I don't know how your tests might differ from mine, they can't be deployed. 
> Although in the original ra2 it was possible.
> 
> By the way, I'm testing this and I came across another bug. Perhaps it's again related to Neutral House. But if you capture and release MCV twice with Yuri, an error occurs. *only in YR mode.
> We need to test this in a game with another player

Make the MCVs deploy into barracks. It works for me. I didn't try ra2 mode tho

**Comment by CCCP84** [comment]
Who needs MCV deployed to the barracks? ))
MCV deploys to the Construction Yard.

**Comment by CnCRAZER** [comment]
> Who needs MCV deployed to the barracks? )) MCV deploys to the Construction Yard.

lol I noticed this because I have a map where you get mcv's from crates that deploy into factories or barracks. you don't get to build a base. I noticed when it deploys into something that's not a construction yard it works
